action.yaml

```diff
@@ -18,24 +18,12 @@ runs:
           documentation_version="${PROJECT_VERSION}-dev"
```

Comment:
If I understand this code correctly: feature branches will overwrite the previous deployed ${PROJECT_VERSION}-dev ?

Reply:
Currently there is a separate workflow for the feature branches / pull requests which does execute this step.
You can see it in the example mentioned here: https://github.com/QActions/java-service-workflow